sanic_routing/tree.py

```diff
@@ -63,8 +64,15 @@ def render(self) -> t.List[Line]:
         output += delayed
         return output
 
+    def apply_offset(self, amt, apply_self=True, apply_children=False):
+        if apply_self:
+            self.offset += amt
+        if apply_children:
+            for child in self.children.values():
+                child.apply_offset(amt, apply_children=True)
+
     def to_src(self) -> t.Tuple[t.List[Line], t.List[Line]]:
```

Comment:
I feel like this whole method needs to be re-thought, and re-written to be less tenuous. But that is a job for a future sprint.

Reply:
I don't disagree. I have some thoughts on this for another day. I think the `apply_offset` method is at least a step in the right direction away from the reindentation process.

Reply:
(in case github didn't make it clear, I put this comment on just line 74, referring to the `to_src()` method in general)

Reply:
I figured. This is probably something we should try and tackle before 21.6.